File: pythonProject/Libro.py
class Libro:
    def __init__(self, id=None, titulo=None, autores=None, genero=None):
        self.id = id
        self.titulo = titulo
        self.autores = autores
        self.genero = genero

    def get_id(self):
        return self.id

    def set_id(self, id):
        self.id = id

    def get_titulo(self):
        return self.titulo

    def set_titulo(self, titulo):
        self.titulo = titulo

    def get_autores(self):
        return self.autores

    def set_autores(self, autores):
        self.autores = autores

    def get_genero(self):
        return self.genero

    def set_genero(self, genero):
        self.genero = genero

    def __str__(self):
        return (f"Libro{{id={self.id}, titulo='{self.titulo}', "
                f"autores={self.autores}, genero='{self.genero}'}}")

File: phpProject/Libro.php
<?php

class Libro {
    private $id;
    private $titulo;
    private $autores;
    private $genero;

    public function __construct($id = null, $titulo = null, $autores = [], $genero = null) {
        $this->id = $id;
        $this->titulo = $titulo;
        $this->autores = $autores;
        $this->genero = $genero;
    }

    public function getId() {
        return $this->id;
    }

    public function setId($id) {
        $this->id = $id;
    }

    public function getTitulo() {
        return $this->titulo;
    }

    public function setTitulo($titulo) {
        $this->titulo = $titulo;
    }

    public function getAutores() {
        return $this->autores;
    }

    public function setAutores($autores) {
        $this->autores = $autores;
    }

    public function getGenero() {
        return $this->genero;
    }

    public function setGenero($genero) {
        $this->genero = $genero;
    }

    public function __toString() {
        return "Libro{id={$this->id}, titulo='{$this->titulo}', autores=" . implode(", ", $this->autores) . ", genero='{$this->genero}'}";
    }
}

?>
